__tests__/app.test.js

```diff
@@ -4,6 +4,7 @@ const seed = require('../db/seeds/seed');
 const testData = require('../db/data/test-data/index');
 const app = require('../app');
 const { response } = require('../app');
+const reviews = require('../db/data/test-data/reviews');
```

Comment:
This doesn't need to be imported in here, it's dealt with on line 4 as part of the index.js file in the test-data/ folder.